src/ComposerRequireChecker/Cli/CheckCommand.php

```diff
@@ -61,8 +61,8 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if (false === $composerJson) {
             throw new \InvalidArgumentException('file not found: [' . $input->getArgument('composer-json') . ']');
         }
-        $this->checkJsonFile($composerJson);
-        $composerData = (new JsonLoader($composerJson))->getData();
+
+        $composerData = (new JsonLoader($composerJson))->getData(); // JsonLoader throws an exception if it cannot load the file
```

Comment:
Please keep the `private` method, rename it, and document this behavior there, rather than on an inline comment